app/uk/gov/hmrc/apinotificationqueue/repository/MongoFormatters.scala

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2017 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.apinotificationqueue.repository
+
+import play.api.libs.json.Json
+
+trait MongoFormatters {
```

Comment:
These aren't necessarily mongo formatters they're just Json formatters, also rather than in a trait we could put them inside of the companion objects of the case classes so this doesn't have to be scattered everywhere.

Reply:
please see above comment. In addition the hmrc MongoRepository that we extend takes a Json formatter for the Mongo entity, which only exists at the repository layer. So I think it makes sense to have separate formatter classes

Reply:
I'm not sure I like the `import JsonFormatters._` in API-sub-fields what does it gain having them all in one trait?

Reply:
I have moved Json formats to companion objects as suggested